homework/jkim237/forms/tables/tables.css

```diff
@@ -0,0 +1,3 @@
+table, th, td {
```

Comment:
There is a template for this homework!

Please move this code to the path `fe-soon/homework/jkim237/20201012/styles/tables.css`